nexus/src/config.rs

```diff
@@ -40,6 +40,13 @@ pub struct ConsoleConfig {
     pub session_absolute_timeout_minutes: u32,
 }
 
+#[derive(Clone, Debug, Default, Deserialize, PartialEq, Serialize)]
+pub struct UpdatesConfig {
+    /** Trusted root.json role for the TUF updates repository. If `None`, accessing the TUF
+     * repository will fail. */
+    pub tuf_trusted_root: Option<PathBuf>,
```

Comment:
Is there any deployment where we might want to leave this out?  (i.e., why not require this?  is it a pain for development?)

Reply:
I imagine this could be temporary, but given all the work going on right now I didn't want to break folks' testing workflows. There should not be production deployments where this is somehow left unconfigured.